<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGuiBattlefield</class>
 <widget class="QMainWindow" name="QtGuiBattlefield">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>792</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtGuiBattlefield</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="buttonCancel">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>660</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonStop">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>690</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>quit</string>
    </property>
   </widget>
   <widget class="QLabel" name="unitID">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>665</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="unitHP">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>695</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="unitMoveD">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>665</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>MoveDistance:</string>
    </property>
   </widget>
   <widget class="QLabel" name="bossID">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>665</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>BossID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="bossHP">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>695</y>
      <width>100</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>BossHP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="unitAttribute">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>695</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Attribute:</string>
    </property>
   </widget>
   <widget class="QLabel" name="bossMoveD">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>665</y>
      <width>150</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>MoveDistance:</string>
    </property>
   </widget>
   <widget class="QLabel" name="bossAttribute">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>695</y>
      <width>190</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>MoveDistance:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonStart">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>660</y>
      <width>93</width>
      <height>58</height>
     </rect>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QLabel" name="tips">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>630</y>
      <width>501</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <italic>true</italic>
     </font>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Tipsï¼šBefore start,you can modify your army position.</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="gameRound">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>685</y>
      <width>100</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Round:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonMove">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>660</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>move</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonWait">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>660</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>wait</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonAttack">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>690</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>attack</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonHeal">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>690</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>heal</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonStart</sender>
   <signal>clicked()</signal>
   <receiver>QtGuiBattlefield</receiver>
   <slot>BtnStart_Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1215</x>
     <y>718</y>
    </hint>
    <hint type="destinationlabel">
     <x>1200</x>
     <y>629</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonStop</sender>
   <signal>clicked()</signal>
   <receiver>QtGuiBattlefield</receiver>
   <slot>BtnStop_Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1082</x>
     <y>741</y>
    </hint>
    <hint type="destinationlabel">
     <x>1036</x>
     <y>736</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>QtGuiBattlefield</receiver>
   <slot>BtnCancel_Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1088</x>
     <y>713</y>
    </hint>
    <hint type="destinationlabel">
     <x>1023</x>
     <y>661</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonMove</sender>
   <signal>clicked()</signal>
   <receiver>QtGuiBattlefield</receiver>
   <slot>BtnMove_Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>827</x>
     <y>710</y>
    </hint>
    <hint type="destinationlabel">
     <x>825</x>
     <y>683</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonWait</sender>
   <signal>clicked()</signal>
   <receiver>QtGuiBattlefield</receiver>
   <slot>BtnWait_Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>922</x>
     <y>714</y>
    </hint>
    <hint type="destinationlabel">
     <x>900</x>
     <y>590</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonAttack</sender>
   <signal>clicked()</signal>
   <receiver>QtGuiBattlefield</receiver>
   <slot>BtnAttack_Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>810</x>
     <y>739</y>
    </hint>
    <hint type="destinationlabel">
     <x>768</x>
     <y>752</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonHeal</sender>
   <signal>clicked()</signal>
   <receiver>QtGuiBattlefield</receiver>
   <slot>BtnHeal_Click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>974</x>
     <y>751</y>
    </hint>
    <hint type="destinationlabel">
     <x>1020</x>
     <y>580</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>BtnStart_Click()</slot>
  <slot>BtnStop_Click()</slot>
  <slot>BtnCancel_Click()</slot>
  <slot>BtnMove_Click()</slot>
  <slot>BtnWait_Click()</slot>
  <slot>BtnAttack_Click()</slot>
  <slot>BtnHeal_Click()</slot>
 </slots>
</ui>
